### PR TITLE
feat(effects-docs): init

### PR DESCRIPTION
**Pull request by cdaringe**
# problem

no docs site

# solution

reaaaal simple docs site

image

### CODE REVIEW

**Comment by cdaringe** [comment]
why pug and not `(docs-toolchain-of-the-day)`?

pug is more or less compact html with a few _very small, very simple_ mechanisms to compose stuff. fully static site.  no dev servers.  no ui runtime.  static sites--solved problem.

File: packages/effects-docs/src/examples.pug [hunk 1]
```diff
@@ -0,0 +1,3 @@
+extends layout.pug
+block main
+  p weeeee
```
Comment (cdaringe) [inline packages/effects-docs/src/examples.pug:3]: very useful file.

File: packages/effects-docs/src/home.pug [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  :markdown-it(linkify langPrefix='highlight-')
+    ```js
+    'use effects'
+    try {
+      await work()
+    } handle (event) {
+      switch event.type {
+        case 'log': {
+          console.log(event)
+          resume
```
Comment (dpchamps) [inline packages/effects-docs/src/home.pug]: As of now, it's `recall` and the `recall` keyword must be passed a value. These are issues, so it's not set in stone. But it should read like this for now:

```suggestion
          recall null
```

related: #6

**Review by dpchamps — APPROVED** [review]
LGTM

I feel like we need a disclaimer somewhere. Something along the lines of, "this is highly experimental. Not ready for prod 😬😬😬." 

I don't know if that belongs in the docs site. But it might be a good place to shout it out.

**Comment by cdaringe** [comment]
> I feel like we need a disclaimer somewhere. Something along the lines of, "this is highly experimental. Not ready for prod 😬😬😬."

yup.  well... there's also no getting started section yet either :)

**Pull request merged**